src/api/index.js

```diff
@@ -89,13 +101,8 @@ let exportObject = {
   getExperiment,
   getAssignments,
   getFilePair,
+  putAnswer,
 };
-if (process.env.NODE_ENV !== 'test') {
-  exportObject = {
-    ...mocks,
-    me,
-  };
-}
 
 // eslint-disable-next-line
```

Comment:
no need for it anymore

Reply:
???

Reply:
I added it because there were mocks.
Now we can safely remove it.
Please take a look how I did it when connected API to UI:
https://github.com/smacker/code-annotation/commit/7340215ba7e96494f264658784e84b793152e3eb#diff-284817f1b6aad47c0ac6d61bd72f474c